pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -51,7 +51,7 @@
 , CoreFoundation
 , CoreServices
 
-# nvim-treesitter dependencies
+  # nvim-treesitter dependencies
```

Comment:
Hmm... I guess these don't need to be in the diff, _but_ this file was run through `nixpkgs-fmt` _and_ the other comments line up to match this.